homeassistant/components/thermostat/ecobee.py

```diff
@@ -26,8 +47,21 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     _LOGGER.info(
         "Loading ecobee thermostat component with hold_temp set to %s",
         hold_temp)
-    add_devices(Thermostat(data, index, hold_temp)
-                for index in range(len(data.ecobee.thermostats)))
+
+    thermostats = []
+    for index in range(len(data.ecobee.thermostats)):
+        thermostats.append(Thermostat(data, index, hold_temp))
+    add_devices(thermostats)
+
+    def send_message_service(service):
+        """Send message to thermostats."""
+        message = service.data['message']
+        for thermostat in thermostats:
+            thermostat.send_message(message)
+
+    hass.services.register(DOMAIN, 'send_message', send_message_service,
```

Comment:
@balloob I hadn't considered it but I think the notify platform would be a better choice.  That being said, I tried a few things and I wasn't able to get the ecobee discovery to successfully call get_service in my notify platform.  I didn't see any other notify platforms using discovery, is it possible?

Thanks!